docs/posts/prepare.md

```diff
@@ -5,20 +5,46 @@ description: Preparing your data
 ---
 # Data format requirements
 
-If your data is in `h5ad` file (from the [`anndata`](https://anndata.readthedocs.io/en/latest/index.html) library) and meets the following requirements, you can go straight to `cellxgene launch`:  
+If your data is in `h5ad` file (from the [`anndata`](https://anndata.readthedocs.io/en/latest/index.html) library) and meets the following requirements, you can go straight to `cellxgene launch`:
 
-- Expression values (raw or normalized) in `anndata.X`  
+- Expression values (raw or normalized) in `anndata.X`
 - At least one embedding (e.g., tSNE, UMAP) in `anndata.obsm`, specified with the prefix `X_` (e.g., by default scanpy stores UMAP coordinates in `anndata.obsm['X_umap']`)
-- A unique identifier for every cell is available in an `anndata.obs` field (you can specify this with the `--obs-names` option)  
-- A unique identifier for every gene is available in an `anndata.var` field (you can specify which field to use with the `--var-names` option)  
+- A unique identifier for every cell is available in an `anndata.obs` field (you can specify this with the `--obs-names` option)
```

Comment:
this is misleading.  What is required is a unique identifier for each cell, which by default will be pulled from the `obs` DataFrame index.  If the index is not unique or does not contain the cell ID, an alternative column can be specified with `--obs-names`.

Reply:
Err. The only reason this line was changed is was that my IDE cleaned up some whitespace. That said, I'll do my best to clarify the docs. Not sure that I have all the context here.